### PR TITLE
Fix coding standards discussed in #86.

### PR DESCRIPTION
**Pull request by finnlewis**

### CODE REVIEW

**Comment by finnlewis** [comment]
Looks like the coding standards are now passing. When the tests finish, @Adnan-cds perhaps you could approve?

File: src/Form/AlertBannerEntityRevisionRevertForm.php [hunk 1]
```diff
@@ -129,7 +129,7 @@ public function submitForm(array &$form, FormStateInterface $form_state) {
   protected function prepareRevertedRevision(AlertBannerEntityInterface $revision, FormStateInterface $form_state) {
     $revision->setNewRevision();
     $revision->isDefaultRevision(TRUE);
-    $revision->setRevisionCreationTime(REQUEST_TIME);
+    $revision->setRevisionCreationTime(\Drupal::time()->getRequestTime());
```
Comment (andybroomfield) [inline src/Form/AlertBannerEntityRevisionRevertForm.php:132]: Should we use dependancy injection here, or is this ok?

**Review by andybroomfield — APPROVED** [review]
Happy for these to merge.

File: src/Form/AlertBannerEntityRevisionRevertForm.php (hunk 1 above)
Comment (finnlewis) [reply src/Form/AlertBannerEntityRevisionRevertForm.php:132]: Sorry, I merged on seeing your approval and then saw your comment. How does dependancy injection work? Can you provide an example?

Comment (Adnan-cds) [reply src/Form/AlertBannerEntityRevisionRevertForm.php:132]: Hi Finn,
Within the create() [method](https://github.com/localgovdrupal/localgov_alert_banner/blob/00d83bfb6c7368771e7387a6d7abe0502f249786/src/Form/AlertBannerEntityRevisionRevertForm.php#L42), you can add the 'datetime.time' service as a class method (say $instance->dateTime).  The REQUEST_TIME constant can then be replaced with ```$this->dateTime->getRequestTime()```.  But don't worry, I will raise another pull request give that this has been merged already.

Comment (andybroomfield) [reply src/Form/AlertBannerEntityRevisionRevertForm.php:132]: In the past I've got deprecation warnings for 'dont use \Drupal methods in classes' however the tests pass.

Comment (Adnan-cds) [reply src/Form/AlertBannerEntityRevisionRevertForm.php:132]: I have added a [pull request](https://github.com/localgovdrupal/localgov_alert_banner/pull/88) to use dependency injection when inserting the request time.  I will assign that to both of you for review if it passes the tests :)

@andybroomfield @finnlewis

**Pull request merged**